MIGRATION_GUIDE.md

```diff
@@ -133,7 +133,7 @@ In case you are not currently calling it, make sure to update your code adding t
 ```kotlin
 class MyActivity : AppCompatActivity() {
 
-  private var lock: Lock? = null
+  private lateinit var lock: Lock
```

Comment:
A simpler syntax for Android apps. "lateinit" means that the variable will be initialized later (e.g. in the on create method). Prevents us from asserting the null scenarios.